### PR TITLE
Fix/PLF-8335: Add dependency of metadata-extractor

### PR DESCRIPTION
**Pull request by lbengayed**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -102,6 +102,7 @@
     <org.apache.poi.version>3.13</org.apache.poi.version>
     <org.apache.poi-ooxml.version>3.13</org.apache.poi-ooxml.version>
     <org.apache.tika.version>1.5</org.apache.tika.version>
+    <com.drewnoakes.version>2.6.2</com.drewnoakes.version>
```
Comment (ahamdi) [inline pom.xml:105]: Why do not use thae latest version 2.14.0 ?

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (lbengayed) [reply pom.xml:105]: this is the old PR, the new PR is https://github.com/Meeds-io/maven-depmgt-pom/pull/11

Comment (ahamdi) [reply pom.xml:105]: Could you close this PR ?

Comment (lbengayed) [reply pom.xml:105]: done!

**Pull request closed without merging**